inspire_matcher/core.py

```diff
@@ -205,5 +207,31 @@ def _compile_nested(query, record):
     return result
 
 
+def _compile_nested_prefix(query, record):
+    path, search_path = query['path'], query['search_path']
+
+    nested_path = search_path.split('.')[0]
+    result = {
+        "query": {
+            "nested": {
+                "path": nested_path,
+                "query": {
+                }
+            }
+        }
+    }
+    value = get_value(record, path)
+    if not value:
+        return
+
+    result['query']['nested']['query'] = {
+        'prefix': {
```

Comment:
instead of `prefix` we should use `match-query-phrase-prefix` https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query-phrase-prefix.html

Reply:
I'd combile everythin in one statement:

```python
result = {

}
```